src/graph.rs

```diff
@@ -1480,6 +1481,83 @@ impl PyGraph {
         Ok(out_dict.into())
     }
 
+    /// Substitute a set of nodes with a single new node.
+    ///
+    /// .. note::
+    ///     The new node is always made to be the target in any edge
+    ///     representation that includes it. For example, the new node
+    ///     will always appear as the target of its edges as returned by
+    ///     :meth:`~retworkx.PyGraph.edge_list`.
```

Comment:
`PyGraph` is undirected, what do you mean here? Just that the tuple `(u, v)` petgraph stores under the covers will always have `node` as `v` so when you call `edge_list()` which just returns those tuples you'll see this? If so, I'm not sure if it's worth documenting that, I'm mostly concerned about potentially having people depend on the behavior as it's mostly a implementation detail not something we want to commit to.

Reply:
> so when you call edge_list() which just returns those tuples you'll see this?

Yup. My thinking behind documenting it this way was because I would expect some users might be confused to find that the original ordering of edge tuples is not preserved (which is done to make our code cleaner (and possibly faster IIRC)). Perhaps I should change this note to warn that the original ordering is not preserved, but then not specify the implementation-specific order?

Reply:
Done in 1959cf8.